guide/src/reference/attributes/on-js-imports/host_binding.md

```diff
@@ -0,0 +1,139 @@
+# `host_binding`
+
+The `host_binding` attribute is the converse of the [`structural`
+attribute](structural.html). It configures how `wasm-bindgen` will generate JS
+glue to call the imported function. The naming here is intended convey that this
+attribute is intended to implement the semantics of the future [host bindings
```

Comment:
I think this is misleading, since both `structural` and `host_binding` can (and will) use the host bindings proposal.